R/utils.R

```diff
@@ -220,7 +220,6 @@ platform_independent_sort <- function(x) x[platform_independent_order(x)]
 #' unlink(tmp)
 #'
 #' # more importantly, extract strings under R>=4 raw strings
```

Comment:
Maybe also reword the example?

Reply:
I originally thought it's still useful to call this out as an R4+ thing, but you're right, eventually this will look out of place, so might as well make the cut now.